src/argilla/client/feedback/dataset.py

```diff
@@ -505,7 +505,12 @@ def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[
                     datasets_api_v1.add_records(
                         client=httpx_client,
                         id=self.argilla_id,
-                        records=[record.dict() for record in self.__new_records[i : i + PUSHING_BATCH_SIZE]],
+                        records=[
+                            record.dict()
+                            for record in tqdm(
```

Comment:
I think that it would be good to pass the `disable` parameter to the `tqdm` function, so user can choose if they want to display the progress bar or not.

A `show_progress` parameter could be added to the `push_to_argilla` method and then pass the value of this param to tqdm `disable` param.

```python
    def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[str, rg.Workspace]] = None, show_progress: bool = False) -> None:
        ...
   ```

Reply:
Thanks for the review. I have updated the PR.